### DISCUSSION TITLE
Getting permissions of GITHUB_TOKEN in workflows

### DISCUSSION THREAD

**Discussion by Piedone** [community/community#37395] [Actions] [answered]
I'd like to know if the current `GITHUB_TOKEN` has `write` access to `PullRequests`. This info is also displayed on the UI for a run:

![image](https://user-images.githubusercontent.com/1976647/198131210-da764866-6d88-4896-aa53-17bd06cc6b00.png)

However, I see no way of getting it programmatically within the workflow. The various contexts don't seem to contain this. Any tips?

**Comment by airtower-luna** [comment, marked as answer]
When you make any API request authorized with the token, the response should contain an `x-oauth-scopes` header that tells you which scopes it has.

**Reply by Piedone** [reply]
I see, thank you. Isn't there a more direct way to do this?

BTW in the meantime, I figured out that I could also check if a PR is for a fork (in my particular use-case, this and the token's permission is equivalent) with `if: github.event.pull_request == '' || github.event.pull_request.head.repo.fork == false`.

**Reply by airtower-luna** [reply]
Yeah, if that's all you need checking if you have a PR from a fork is a lot simpler. :slightly_smiling_face: 

And apparently not, that's the method recommended in the docs: https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps

**Reply by Piedone** [reply]
I see, thank you. The fork approach is not ideal and what I really need is the permission check. But I'll consider the API request one.
